config.ru

```diff
@@ -0,0 +1,2 @@
+require './i-scream-arcade-microservice-app'
+run IScreamMicroservice
```

Comment:
This is the file that runs the microservice, needed to run rspec (at least this is what worked for me)

Reply:
Cool. That's good to know